seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -5327,710 +5327,6 @@
         "<h4>Instrucciones</h4>",
         "Usa <code>/\\S/g</code> para contar el número de caracteres que no están en blanco en <code>testString</code>."
       ]
-    },
```

Comment:
I thought that was it but I needed to remove the comma at the end. And the way GitHub highlights things makes it look like I've deleted it but the matching bracket is [here](https://github.com/FreeCodeCamp/FreeCodeCamp/pull/8267/commits/efde04ea988317ea2c1a1ce86a26088b25662cb3#diff-5fbffcdf48f7976dd3b0216c387eebc2R5330) at the end of the file.